cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,62 @@
-describe('', () => {
-  before(() => {
+/// <reference types='cypress' />
 
+describe('Article flow', () => {
+  let user;
+  let article;
+
+  beforeEach(() => {
+    cy.visit('/');
+    cy.task('generateUser').then((generatedUser) => {
+      user = generatedUser;
+      cy.login(user.email, user.username, user.password);
+      cy.getCookie('auth').should('exist');
+      cy.reload();
+      cy.task('generateArticle').then((generatedArticle) => {
+        article = generatedArticle;
+      });
+    });
   });
 
-  it('', () => {
+  it('should allow user to create an article', () => {
+    cy.contains('.nav-link', 'New Article').click();
+    cy.url().should('contain', '/editor');
+
+    cy.findByPlaceholder('Article Title').type(article.title);
+    cy.findByPlaceholder(`What's this article about?`)
+      .type(article.description);
+    cy.findByPlaceholder('Write your article (in markdown)')
+      .type(article.body);
+    cy.findByPlaceholder('Enter tags')
+      .type(`${article.tagList}{enter}`);
```

Comment:
Ensure that the `tagList` is correctly formatted as a string with tags separated by commas or spaces, as required by the input field. The current implementation might not work if `tagList` is an array.